cmd/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/filecoin-project/go-data-transfer v1.14.0
-	github.com/filecoin-project/go-legs v0.2.7
+	github.com/filecoin-project/go-legs v0.2.8-0.20220208004129-dc48d0d70036
```

Comment:
Is the plan to depend on a release of go-legs before this PR is merged?

Reply:
yeah, once we merge the legs PR we'll want to cut a release and rebase